packages/uniforms/src/JSONSchemaBridge.js

```diff
@@ -0,0 +1,229 @@
+import invariant from 'fbjs/lib/invariant';
+import joinName  from './joinName';
+
+import Bridge from './Bridge';
+
+let Ajv;
+try {
+    const r = require; // Silence Meteor missing module warning
+    Ajv = r('ajv');
+} catch (_) { /* Ignore it. */ }
+
+const resolveRef = (referance, schema) => {
+    invariant(
+        referance.startsWith('#'),
+        'Reference is not an internal reference, and only such are allowed: "%s"',
+        referance
+    );
+
+    const resolvedReference = referance
+        .split('/')
+        .filter(part => part && part !== '#')
+        .reduce((definition, next) => definition[next], schema);
+
+    invariant(resolvedReference, 'Reference not found in schema: "%s"', referance);
+
+    return resolvedReference;
+};
+
+export default class JSONSchemaBridge extends Bridge {
+    constructor (schema) {
```

Comment:
_(as we discussed it internally)_

How about make the validator configurable? In this case, it'd be `new JSONSchemaBridge(schema, validator)`. It'll make it more configurable and we'd get rid of another dependency and/or a problem with `ajv` options.